pki.spec

```diff
@@ -52,15 +52,14 @@ Source: https://github.com/dogtagpki/pki/archive/v%{version}%{?_phase}/pki-%{ver
 ################################################################################
 
 %define java_home /usr/lib/jvm/jre-openjdk
+%define java_devel java-devel
+%define java_headless java-headless
 
 %if 0%{?fedora} && 0%{?fedora} >= 33
-%define min_java_version 1:1.8.0
-%define java_devel java-1.8.0-openjdk-devel
-%define java_headless java-1.8.0-openjdk-headless
+%define min_java_version 1:11
+%define is_java_11 1
```

Comment:
It's not really a problem, but semantically this is probably incorrect. The `min_java_version` implies that the version can be greater than 11, but then `is_java_11` will be set to 1. See my other comment below.